schema/Dockerfile

```diff
@@ -0,0 +1,38 @@
+#-------------------------------------------------------#
+# First stage: image to build and test node application #
+#-------------------------------------------------------#
+FROM node:14-alpine as build-stage
+
+WORKDIR /build
+
+# Copy package*.json files first in order to make best use of docker layer caching
+COPY ./package*.json ./
+
+# npm clean slate install to get reproducible builds and quicker installs
+RUN npm ci
+
+# copy rest of the files
+COPY ./src ./src
```

Comment:
pls copy src last since it changes most often (utilizes docker layer caching better)